worlds/_bizhawk/__init__.py

```diff
@@ -113,7 +113,12 @@ async def send_requests(ctx: BizHawkContext, req_list: typing.List[typing.Dict[s
     """Sends a list of requests to the BizHawk connector and returns their responses.
 
     It's likely you want to use the wrapper functions instead of this."""
-    return json.loads(await ctx._send_message(json.dumps(req_list)))
+    responses = json.loads(await ctx._send_message(json.dumps(req_list)))
+    for response in responses:
+        if response["type"] == "ERROR":
+            raise ConnectorError(response["err"])
```

Comment:
Is it ever expected that there may be multiple errors returned at once? If so, starting with Python 3.11 ExceptionGroup could be used to capture them all.

Reply:
It's possible. Sort of taking a guess here at the proper way to do this. Tested multiple errors getting returned on 3.11 and 3.10. Both worked as expected.

Reply:
Hm. It failed a check though, even being behind the if statement. I'm unsure how to address that.

Reply:
you should be able to suppress it via # noqa

Reply:
Neat, appears to have done it.